index.js

```diff
@@ -236,7 +237,13 @@ exports.request = function (options, callback) {
 
     if (options.file) {
         cmd = 'cat';
-        args = [options.file];
+        var rootDirectory = path.resolve(process.cwd(), './');
```

Comment:
`process.cwd()` is already called and defined above as variable `cwd` ([Line 5](https://github.com/node-js-libs/curlrequest/blob/master/index.js#L5)) so you should just pass `cwd` to `path.resolve()` instead of calling the function again.

```javascript
var rootDirectory = path.resolve(process.cwd(), './');
```

:point_down: 

```javascript
var rootDirectory = path.resolve(cwd, './');
```

Reply:
Missed that! Thanks for pointing it out